edx_sga/sga.py

```diff
@@ -165,9 +166,9 @@ def student_upload_max_size(cls):
         """
         returns max file size limit in system
         """
-        return getattr(
-            settings, "STUDENT_FILEUPLOAD_MAX_SIZE", cls.STUDENT_FILEUPLOAD_MAX_SIZE
-        )
+        return configuration_helpers.get_value('STUDENT_FILEUPLOAD_MAX_SIZE', getattr(
```

Comment:
It seems this file uses double quotes, so could you please use double quotes and avoid mixing double and single quotes. @Nekenhei